share/site/duckduckgo/app.tx

```diff
@@ -0,0 +1,125 @@
+                    </noscript>
+                    <img class="js-lazysvg no-js__hide" src="" data-src="/assets/app/btn-icon-android" />
+                    Android
+                </a>
+            </div>
+        </div>
+        <div class="app__screens">
+            <div class="app__device app__device--left app__device--android">
+                <noscript>
+                    <img class="app__device__frame js-lazyload no-js__hide" src="/assets/app/screens/android-device-grey.png"/>
+                </noscript>
+                <img class="app__device__frame js-lazyload no-js__hide" src="" data-src="/assets/app/screens/android-device-grey.png"/>
+
+                <noscript>
+                    <img class="app__device__screen js-lazyload no-js__hide" src="/assets/app/screens/android-screenshot-favorites.jpg"/>
+                </noscript>
+                <img class="app__device__screen js-lazyload no-js__hide" src="" data-src="/assets/app/screens/android-screenshot-favorites.jpg"/>
+            </div>
+            <div class="app__device app__device--android">
+                <noscript>
+                    <img class="app__device__frame js-lazyload no-js__hide" src="/assets/app/screens/android-device-grey.png"/>
+                </noscript>
+                <img class="app__device__frame js-lazyload no-js__hide" src="" data-src="/assets/app/screens/android-device-grey.png"/>
+
+                <noscript>
+                    <img class="app__device__screen js-lazyload no-js__hide" src="/assets/app/screens/android-screenshot-feed.jpg"/>
+                </noscript>
+                <img class="app__device__screen js-lazyload no-js__hide" src="" data-src="/assets/app/screens/android-screenshot-feed.jpg"/>
+            </div>
+            <div class="app__device app__device--right app__device--android">
+                <noscript>
+                    <img class="app__device__frame js-lazyload no-js__hide" src="/assets/app/screens/android-device-grey.png"/>
+                </noscript>
+                <img class="app__device__frame js-lazyload no-js__hide" src="" data-src="/assets/app/screens/android-device-grey.png"/>
+
+                <noscript>
+                    <img class="app__device__screen js-lazyload no-js__hide" src="/assets/app/screens/android-screenshot-story.jpg"/>
+                </noscript>
+                <img class="app__device__screen js-lazyload no-js__hide" src="" data-src="/assets/app/screens/android-screenshot-story.jpg"/>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="app app--features blk blk--alt">
+    <div class="app--features__cw cw--c">
+		<div class="app__row">
+			<div class="app__key third half--screen-m">
+				<noscript>
+					<img class="app__key__icon" src="/assets/about/icons/search-alt.png" />
+				</noscript>
+				<img class="app__key__icon js-lazysvg no-js__hide" src="" data-src="/assets/about/icons/search-alt">
```

Comment:
I wonder if we can figure out a better way to handle this. It sucks that we have to send the same image markup down twice...